js/hue-service.js

```diff
@@ -1,18 +1,19 @@
 (function() {
     'use strict';
 
-    function HueService($http) {
+    function HueService($http, $translate) {
         var service = {};
-
         //Updates a group of Hue lights (Assumes that one group is configured)
         //You can change the group to 0 to perform the updates to all lights
         service.performUpdate = function(spokenWords) {
             //deturmine the updates that we need to perform to the lights
             var update = deturmineUpdates(spokenWords.toLowerCase().split(" "));
+            // Deturmine which light to swith on/off
+            var light = deturmineLight(spokenWords.toLowerCase().split(" "));
```

Comment:
Seems redundant to `spokenWords.toLowerCase().split(" ")` twice. Can you create a var for this that can be consumed by both functions?